### DISCUSSION TITLE
Add option to hide the link below the quote that opens up the extension page

### DISCUSSION THREAD

**Discussion by 0xTaoL** [jordwest/news-feed-eradicator#436] [Ideas]
Currently, it is too frictionless to click on that link when I am already on the site to show the feed temporarily. An option to hide that link might increase the deterrance to scroll.

**Comment by gregorylearns** [comment]
I've been trying to figure this out since forever! It has become second nature to click the link to temporarily disable. I have found a workaround with uBlock Origin (which I also use in conjunction). Not super familiar with javascript and i keep failing to build the extension when I try it.

it seems to be connected to this 'nfe-info-link' element

I go into element picker mode in uBlock origin and block the link directly.


Here's my filter settings on Facebook on uBlock origin settings:
```
! Apr 24, 2025 https://www.facebook.com
www.facebook.com##.nfe-info-link > span
```
